openfisca_us/variables/demographic/person.py

```diff
@@ -17,7 +17,7 @@ class person_weight(Variable):
 
 
 class age(Variable):
-    value_type = int
+    value_type = float
```

Comment:
@nikhilwoodruff fyi, since the rules are based on age more fine than int, e.g. 1.5 years